lib/matplotlib/tests/test_datetime.py

```diff
@@ -276,11 +276,38 @@ def test_quiverkey(self):
         fig, ax = plt.subplots()
         ax.quiverkey(...)
 
-    @pytest.mark.xfail(reason="Test for scatter not written yet")
     @mpl.style.context("default")
     def test_scatter(self):
-        fig, ax = plt.subplots()
-        ax.scatter(...)
+        mpl.rcParams["date.converter"] = 'concise'
+        base = datetime.datetime(2005, 2, 1)
+        dates = [base + datetime.timedelta(hours=(2 * i)) for i in range(732)]
```

Comment:
We probably don't need quite so long an array; maybe something like 10 items is good enough.